docs/reference/ml/anomaly-detection/aggregations.asciidoc

```diff
@@ -11,6 +11,11 @@ distributes these calculations across your cluster. You can then feed this
 aggregated data into the {ml-features} instead of raw results, which
 reduces the volume of data that must be considered while detecting anomalies.
 
+The aggregation you create in a {dfeed} needs to have the same name as that of 
```

Comment:
These seem like quite detailed requirements and I'd expect them to appear lower in the page where we're actually providing implementation examples. For example:

> The aggregation you create... the field it operates on.

I'd suggest putting this right near (or as a footnote on) the PUT _ml/datafeeds/datafeed-farequote example.

> The job analyzes only the fields that you provide in the configuration.

I'd suggest putting this right near (or as a footnote on) the PUT _ml/anomaly_detectors/farequote example.

Reply:
Thanks! I moved the explanation to the implementation examples as suggested.